IA-1884 Instance: save the panningId send by the mobile app

olethanh opened the pull request:
Mobile app was sending the planning id the submission was for but it was not saved server side.

thus we couldn't link back.
Fix IA-1884

## Self proof reading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [NA] Are my typescript files well typed
- [NA] Did I add translations
- [x] My migrations file are included
- [x] Are there enough tests

## Changes

Add field to add the info, add test

## How to test

Create a planning and send a submission from the app, check that we have the info in DB (it's not exposed in the API yet)

quang-le reviewed (approved):
It's basically a revert, so I don't see a problem

olethanh commented:
test in local and it fix the problem

olethanh committed:
IA-1884 Instance: save the panningId send by the mobile app

Mobile app was sending the planning id the submission was for but it was not saved server side
Fix IA-1884

olethanh committed:
Merge remote-tracking branch 'origin/main' into IA-1884-save_mobile_planning_id

olethanh committed:
Merge migrations
